.github/workflows/release.yml

```diff
@@ -173,20 +173,97 @@ jobs:
           path: |
             dist/*.rpm
             !dist/*PAYG*.rpm
-  api-tests:
+
+  test-controller:
     needs: goreleaser
     runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.scope.outputs.matrix }}
+      gd_tag: ${{ steps.versions.outputs.gd_tag }}
+      versions: ${{ steps.versions.outputs.versions }}
+
+    steps:
+      - name: set version
+        id: versions
+        env:
+          GW_OR_DASH: ${{ (endsWith(github.repository, 'tyk') || endsWith(github.repository, 'tyk-analytics')) && 'yes' }}
+          BASE_REF: ${{ startsWith(github.event_name, 'pull_request') && github.base_ref || github.ref }}
+          TAGS: ${{ needs.goreleaser.outputs.tags }}
+          # Cover pull_request_target too
+          PR: ${{startsWith(github.event_name, 'pull_request') && 'yes' }}
+          LTS_PR: ${{(startsWith(github.event_name, 'pull_request') && endsWith(github.base_ref, '-lts')) && 'yes' }}
+          TAG: ${{startsWith(github.ref, 'refs/tags') && 'yes' }}
```

Comment:
@alephnull I can't figure out where this vars are being used. Should we removed or is the logic not finished?

Reply:
TAGS is not used as we do not (yet) have the test matrix that should be run for tags. The logic is complete.